presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/Table.java

```diff
@@ -56,23 +56,24 @@ public OptionalInt getBucketCount()
     }
 
     @Override
-    public int hashCode()
-    {
-        return Objects.hash(tableId, bucketCount);
```

Comment:
The commit message is misleading. It should be something like "Include distributionId in Table equals/hashCode"